docker-image/v1.8/debian-elasticsearch7/Gemfile

```diff
@@ -14,8 +14,9 @@ gem "fluent-plugin-record-modifier", "~> 2.0.0"
 gem "fluent-plugin-detect-exceptions", "~> 0.0.12"
 gem "fluent-plugin-rewrite-tag-filter", "~> 2.2.0"
 gem "elasticsearch", "~> 7.0"
-gem "fluent-plugin-elasticsearch", "~> 3.7.1"
-gem "elasticsearch-xpack", "~> 7.4.0"
+gem "fluent-plugin-elasticsearch", "~> 4.0"
+gem "elasticsearch-xpack", "~> 7.0"
```

Comment:
elasticsearch-xpack shouldn't be downgraded.
Your patch sometimes contains contaminated upgrading/downgrading.

Reply:
No, I just harmonized the elasticsearch-xpack version number with the one of elasticsearch plugin. I will make an other PR for this.

Reply:
Make sense.